test/index.spec.js

```diff
@@ -49,6 +49,16 @@ describe('react', () => {
     });
 });
 
+describe('lodash', () => {
+    it('should disable react if options.lodash is disabled', () => {
```

Comment:
should disable ~~react~~ *lodash* if options.lodash is disabled

Reply:
Oops, fixed.